src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -85,9 +85,21 @@ sortObjects(ExecutionSpace const &space, ViewType &view)
 
   Kokkos::MinMaxScalar<ValueType> result;
   Kokkos::MinMax<ValueType> reducer(result);
-  parallel_reduce("ArborX::Sorting::find_min_max_view",
-                  Kokkos::RangePolicy<ExecutionSpace>(space, 0, n),
-                  Kokkos::Impl::min_max_functor<ViewType>(view), reducer);
+  parallel_reduce(
```

Comment:
Are we doing `using namespace Kokkos;` somewhere? Shouldn't this be `Kokkos::parallel_reduce`?

Reply:
It finds it via ADL because of the execution policy.   But yes we should have qualified the call for consistency.